pkg/auth/tokenparser.go

```diff
@@ -0,0 +1,83 @@
+package auth
+
+import (
+	"errors"
+	"log"
+
+	"github.com/codeready-toolchain/registration-service/pkg/configuration"
+
+	jwt "github.com/dgrijalva/jwt-go"
+)
+
+// TokenClaims represents access token claims
+type TokenClaims struct {
+	Name          string         `json:"name"`
+	Username      string         `json:"preferred_username"`
+	GivenName     string         `json:"given_name"`
+	FamilyName    string         `json:"family_name"`
+	Email         string         `json:"email"`
+	EmailVerified bool           `json:"email_verified"`
+	Company       string         `json:"company"`
+	SessionState  string         `json:"session_state"`
```

Comment:
We don't need SessionState. Also it's more implementation detail. It can be changed in RHD anytime.

Reply:
fixed here: 24c8518f9df59bbb108003c9d1ad6da9adb0e4cd